ccdImports/configFiles/CCD_Probate_Backoffice/ComplexTypes.json

```diff
@@ -55,7 +55,7 @@
   {"LiveFrom": "01/01/2017", "ID": "ExecutorActingForTrustCorp", "ListElementCode": "additionalExecForenames", "FieldType": "Text", "ElementLabel": "First name(s)", "SecurityClassification": "Public"},
   {"LiveFrom": "01/01/2017", "ID": "ExecutorActingForTrustCorp", "ListElementCode": "additionalExecLastname", "FieldType": "Text", "ElementLabel": "Last name(s)", "SecurityClassification": "Public"},
   {"LiveFrom": "01/01/2017", "ID": "ExecutorActingForTrustCorp", "ListElementCode": "additionalExecutorTrustCorpPosition", "FieldType": "Text", "ElementLabel": "Name of their position within the trust corporation as per the resolution", "SecurityClassification": "Public"},
-  {"LiveFrom": "01/01/2017", "ID": "ExecutorActingForTrustCorp", "ListElementCode": "additionalExecAddress", "FieldType": "AddressUK", "ElementLabel": "What is their address?", "SecurityClassification": "Public"},
+  {"LiveFrom": "01/01/2017", "ID": "ExecutorActingForTrustCorp", "ListElementCode": "additionalExecAddress", "FieldType": "AddressUK", "DisplayContext": "MANDATORY", "ElementLabel": "What is their address?", "FieldShowCondition": "additionalExecutorTrustCorpPosition=\"HIDE_FIELD\"", "SecurityClassification": "Public"},
```

Comment:
"FieldShowCondition": "additionalExecutorTrustCorpPosition=\"HIDE_FIELD\"" - we could possibly replace with  [STATE]=\"NeverShow\"; we use this in CaseEventToFields.json, not sure thos if also available to ComplexTypes.json